src/LazyLoadImage/LazyLoadImage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import type { LazyLoadImageProps } from "./LazyLoadImage.types";
+
+// TODO: add tests in browser once we are done with the migration to cypress
+
+/**
+ * Lazy load wrapper for HTML image brackets. Used for lazy loading images.
+ * @param LazyLoadImgProps The input object of the component
+ * @param LazyLoadImgProps.src string Source of the image
+ * @param LazyLoadImgProps.alt string Alternative text for the image
+ * @param LazyLoadImgProps.ImgProps object ImgProps Any other prop of the <img> component
+ */
+export default function LazyLoadImage({
+  src,
+  alt,
+  ImgProps
+}: LazyLoadImageProps) {
+  // state to track if the image is visible
+  const [isVisible, setIsVisible] = useState(false);
+
+  // reference to the image element
+  const imgRef = useRef(null);
+
+  // when the image is in view, set isVisible to true
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      entries => {
+        if (entries[0].isIntersecting) {
+          setIsVisible(true);
+          observer.disconnect();
+        }
+      },
+      { threshold: 0.1 }
+    );
+    if (imgRef.current) {
+      observer.observe(imgRef.current);
+    }
+    return () => observer.disconnect();
+  }, []);
+
+  // render the image
+  return (
+    <img
+      alt={alt || ""}
+      ref={imgRef}
+      loading="lazy"
+      src={isVisible ? src : undefined}
+      style={{ display: isVisible ? "block" : "none" }}
```

Comment:
i think this is not correct, we are passing style two time and something broken here. Shold be like this
```suggestion
      style={{
        ...ImgProps?.style,
        display: isVisible ? "block" : "none",
        width: "100%"
      }}
```

Reply:
Generally I agree that the style should be spread, but I'm not so sure we want to set the width to 100%, I think that comes down to the usage of the component. This is a general lazy loader for any images, I don't think we should restrict the styling other than what is strictly necessary (which is the `display` prop)

Reply:
@dmbarry86 any opinion about this?

Reply:
I agree, this component is simple a wrapper for <img>. This component shouldn't be making any decisions on styling, other than what it needs for the display hiding during load.